Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/app/app.component.html

```diff
@@ -16,10 +19,13 @@ <h1> Elysia Range Simulator </h1>
       <br>
       <label>Battery Percentage (%):</label>
       <br>
-      <input class="form-control"  [ngClass]="{
-	  'invalid-field': inputBattery.invalid && (inputBattery.dirty || inputBattery.touched),
-	  'valid-field': inputBattery.valid && (inputBattery.dirty || inputBattery.touched)
-	  }" name="battery" formControlName="inputBattery" type="number" placeholder="Battery" (keyup)="onKeyBattery($event)" >
+      <input [ngClass]="{
+		  'invalid-field': inputBattery.invalid && (inputBattery.dirty || inputBattery.touched),
+		  'valid-field': inputBattery.valid && (inputBattery.dirty || inputBattery.touched),
+		  'form-control': inputBattery.prestine
+		}"
+		name="battery" formControlName="inputBattery" type="number" placeholder="Battery" 
+	(keyup)="onKeyBattery($event)" >
```

Comment:
Also these lines are supposed to be indented, not spaced, I just added spaces for the comment . So it should look something like this:

![image](https://user-images.githubusercontent.com/29678619/105439935-fab8f180-5c22-11eb-9d52-a6d35955f585.png)